README.md

```diff
@@ -4,19 +4,26 @@
 [![Crates.io](https://img.shields.io/crates/v/autometrics.svg)](https://crates.io/crates/autometrics)
 [![Discord Shield](https://discordapp.com/api/guilds/950489382626951178/widget.png?style=shield)](https://discord.gg/kHtwcH8As9)
 
-Autometrics provides a macro that makes it easy to instrument any function with the most useful metrics: request rate, error rate, and latency. It then uses the instrumented function names to generate powerful Prometheus queries to help you identify and debug issues in production.
+Autometrics provides a macro that makes it trivial to instrument any function with the most useful metrics: request rate, error rate, and latency.
```

Comment:
the one thing I would add here (which probably also should be added to the main/meta repo is: "what problem are we solving?" I think we're 50% there but we mostly talk about it from the solution space, so; "Instrumenting your code and coming up with the right metrics to track the performance of you application is a tedious and painful task. Autometrics is a../provides a macro...."

Reply:
For the problem statement, I'd want to capture both the instrumentation side and the querying side, which is a little difficult to do succinctly. I'd probably say something like:

> Metrics are a powerful tool for understanding the health and performance of your code in production. However, it is tedious to figure out what metrics to track and hard to query the data to make sense of it.
>
> Autometrics provides a macro that makes it trivial to instrument any function with the most useful metrics: request rate, error rate, and latency. It then generates powerful Prometheus queries based on your function details to help you quickly identify and debug issues in production.

That version is missing the "thin layer on top of existing Prometheus and OpenTelemetry libraries" and the point about standardization, but maybe that's okay?

Reply:
I’d be keen to get the standardization point across tbh, it’s such a powerful concept. 

Autometrics provides a macro that makes it trivial to instrument any function with the most useful metrics: request rate, error rate, and latency. It standardizes the metrics through function names and then generate…